session.go

```diff
@@ -160,3 +160,30 @@ func (s *session) sendCommand(cmdType command, cmd string) Packet {
 	results := s.readNBytes(int(r1))
 	return newPacket(int(rep), results)
 }
+
+func (s *session) sendvoidCommand(cmdType command, cmd string) error {
```

Comment:
Is this function needed? If a `cmdvoidEval` is passed into `sendCommand` maybe we could just not perform the r1<= 0 check? 
If this function is needed, maybe we can reduce the amount of duplication between `sendCommand` and `sendvoidCommand`?